CMakeLists.txt

```diff
@@ -69,7 +69,7 @@ MARK_AS_ADVANCED(STORM_QVBS_ROOT)
 
 ## Configuring dependencies
 set(STORM_CARL_GIT_REPO "https://github.com/moves-rwth/carl-storm.git" CACHE STRING "For fetching carl: which git repo to use")
-option(STORM_CARL_GIT_TAG "For fetching carl: which git tag to use" "14.29")
+option(STORM_CARL_GIT_TAG "For fetching carl: which git tag to use" "master")
```

Comment:
For stability reasons, I would prefer not to use the master branch. Then issues on the master do not immediately affect users. Instead of a concrete tag we could also use `stable`.

Reply:
Yes, I think this is a good way. In additional, we can add some test to the CI which always uses the carl master branch. In the current version, we were building Storm against the latest ci Docker image of carl and against the latest release of carl. Maybe we can try the same again?